codalab/worker_manager/main.py

```diff
@@ -118,7 +120,10 @@ def main():
         )
 
     manager = worker_manager_types[args.worker_manager_name](args)
-    manager.run_loop()
+    try:
+        manager.run_loop()
+    except LoginPermissionError:
+        print('All attempts have failed. Please double check your login credentials.')
```

Comment:
I think you mentioned that it will infinitely prompt the user to retry. Can you clarify for what case it will hit this logic?

Reply:
Currently it's not hitting this point. The reason I didn't delete it is that if we are going to limit the number of retries, or give the user an option to quit, then we can use this message.

Reply:
If we are not going to do that, I'll delete it.